packaging/machine-learning-api.spec

```diff
@@ -24,7 +24,7 @@
 %define		tensorflow2_gpu_delegate_support 0
 %endif
 
-%ifnarch %arm aarch64 x86_64 i586 i686 %ix86
```

Comment:
Let's not enable nnfw for RISC-V until nnfw ( https://github.com/Samsung/One ) is ready for RISC-V

Reply:
Indeed, for now, supporting everything is not our main priority, so I dropped the ifnarch for riscv64.